demo-shell/src/app/components/process-service/process-service.component.ts

```diff
@@ -403,6 +404,10 @@ export class ProcessServiceComponent implements AfterViewInit, OnDestroy, OnInit
         this.reloadProcessFilters();
     }
 
+    onStartProcessError(event: any) {
+        this.notificationService.openSnackMessage(event.response.body.message);
```

Comment:
can we use the https://github.com/Alfresco/alfresco-ng2-components/blob/develop/lib/core/notifications/services/notification.service.ts#L74 ?